src/components/NamedLink/NamedLink.js

```diff
@@ -58,17 +58,6 @@ export const NamedLinkComponent = props => {
   );
 };
 
-NamedLinkComponent.defaultProps = {
```

Comment:
Default props will be removed in future versions of React. I couldn't upgrade React to the latest v18 versions because it added a lot of annoying warnings for each use of `defaultProps` and `propTypes`:
See: https://github.com/facebook/react/pull/16210